content/guides/developer-guides/delfin/_index.md

```diff
@@ -9,7 +9,7 @@ tags: ["developer guide", "delfin"]
 # Driver Development Guide
 
 
-Delfin is an Infrastructure Management framework under SODA foundation projects for Heterogeneous storage backends (cloud-enabled, offline-storage, etc.) for management, status collection, telemetry and alerting.
+Delfin is an Infrastructure Management framework under SODA foundation projects for Heterogeneous storage backends (cloud-enabled, offline-storage, etc.) for management, status collection and alerting.
```

Comment:
nit: The use of the word `telemetry` here does not refer to the telemetry feature. Delfin does provide telemetry information.

Reply:
Fixed.